### ISSUE TITLE
Please clarify how to use this for scripting

### ISSUE THREAD

**Issue by JonTheNiceGuy** [ffainelli/faifa#6]
This query is based on the ubuntu packaged version in 14.04, but if I'm using it wrong, or a later version might help me, please let me know.

I see that I can use faifa -m and then enter various frames to send, but running

    echo 0x0014 | faifa

or

    faifa 0x0014

This just returns

    Faifa for HomePlug AV (SVN revision 82)

How can I get it to return the response from the frame that I've requested

    Frame: Central Coordination Discover List Request (0x0014)
    Dump:
    Frame: Central Coordination Discover List Confirm (0015), HomePlug-AV Version: 1.1
    Number of Stations: 4
    <snip>

**Comment by 4920441** [comment]
Hi,

that's the same question I have... I am trying to get RX / TX Rate and SNR maybe the tonemap exported by a script somehow... any thoughts?

Thanks a lot for your great work so far :-) 

Cheers,

**Comment by ffainelli** [comment]
Well, I have to look at this again, I do not think the current design was really well tested, let me get back to you guys in a few days hopefully with working code ;)

**Comment by pkochnov** [comment]
Here is script around faifa I use at home.
[plc.sh.txt](https://github.com/ffainelli/faifa/files/243559/plc.sh.txt)

**Comment by ffainelli** [comment]
Your scripts is actually looking pretty neat, did you want to included it in the faifa sources?

**Comment by pkochnov** [comment]
Hi, I wanted to show guys the example how to use faifa in scripting. It you think it will be helpful for everyone - go ahead and include it in the faifa sources. I'm glad to help. :)

**Comment by JonTheNiceGuy** [comment]
Sorry for the delay - work/life balance got pretty out of whack.

I've tried using the plcc.sh.txt script to probe my network, but I'm still getting nothing. I've pulled the pieces apart on that script, and found I'm doing things "the right way" but they're not firing properly.

When I perform `echo 0014 | faifa -i enp0s25` I get no response, but if I then change that to `echo 0014 | faifa -i enp0s25 -m` it works and then loops through the menu. I suspect that if I were to cause an exit before the second time the menu was triggered, it would probably do what I want it to do. But I'm guessing it's not quite that easy!
